packages/web-components/src/checkbox/checkbox.ts

```diff
@@ -478,4 +492,18 @@ export class Checkbox extends BaseCheckbox {
    */
   @attr
   public size?: CheckboxSize;
+
+  /**
+   * Handles changes to size attribute custom states
+   * @param prev - the previous state
+   * @param next - the next state
+   */
+  public sizeChanged(prev: CheckboxSize | undefined, next: CheckboxSize | undefined) {
+    if (prev) {
+      toggleState(this.elementInternals, `${prev}`, false);
+    }
+    if (next) {
+      toggleState(this.elementInternals, `${next}`, true);
+    }
```

Comment:
Since `prev` and `next` are guarded string values, could these pass the vars directly rather than wrapping it in a template literal?
```suggestion
    if (prev) {
      toggleState(this.elementInternals, prev, false);
    }
    if (next) {
      toggleState(this.elementInternals, next, true);
    }
```

Reply:
Good call, I'll make a note to go and fix the other guarded values as well once we get through the full set.